### DISCUSSION TITLE
Applying for Teacher Status with Email that was previously registered as Student status

### DISCUSSION THREAD

**Discussion by awwsmith** [community/community#52099] [GitHub Education]
I was previously a University student and registered for the GitHub student benefits. Since then, I have been hired as faculty at the same University and retained the same email I used as a student. As a result, my application for Teacher Status are rejected as the .edu email I am using has already received the student benefits.

How can I move from student to teacher status?

**Comment by samus-aran** [comment]
Hi @awwsmith 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by samus-aran** [reply]
Hi @awwsmith 👋 

You should be able to re-apply using the faculty application. Let us know how you get on 👍
